scripts/plugins/storage/sqlite/init.sql

```diff
@@ -680,6 +687,10 @@ INSERT INTO foglamp.scheduled_processes ( name, script ) VALUES ( 'North Statist
 --
 INSERT INTO foglamp.scheduled_processes ( name, script ) VALUES ( 'North_Readings_to_PI',   '["tasks/north_c"]' );
 
+-- South Tasks - C code
+--
+INSERT INTO foglamp.scheduled_processes ( name, script ) VALUES ( 'dht11',   '["services/c-south"]' );
```

Comment:
Better to use:

"services/south_c"
same as "tasks/north_c"

Try keeping the same naming convention

Reply:
ok sure.